plugins/modules/lxd_container.py

```diff
@@ -383,26 +385,6 @@
           mode: pull
           alias: "22.04"
         target: node02
-
-# An example for creating a virtual machine
-- hosts: localhost
-  connection: local
-  tasks:
-    - name: Create container on another node
-      community.general.lxd_container:
-        name: new-vm-1
-        type: virtual-machine
-        state: started
-        ignore_volatile_options: true
-        wait_for_ipv4_addresses: true
-        profiles: ["default"]
-        source:
-          protocol: simplestreams
-          type: image
-          mode: pull
-          server: https://images.linuxcontainers.org
-          alias: debian/11
-        timeout: 600
```

Comment:
This is the only example showing how to create a virtual machine with the module. Instead of deleting the whole example, how about simply replacing the broken URL with `...` and a note that you need to provide an URL there?

Reply:
ah ok, then the doc should probably reflect this, like:
Is there any other URL we could add instead of `...`? I'm currently not aware of any other site, maybe someone else knows one?
I think this would make things more clear.

Reply:
I'm not aware of any since I don't use lxd at all.